src/app/modules/item/helpers/item-children.ts

```diff
@@ -0,0 +1,10 @@
+import { ItemChild } from '../http-services/get-item-children.service';
+
+export interface ItemChildWithAdditions extends ItemChild {
```

Comment:
Also, I am not sure it is worth expending the whole "ItemChild". Probably you just use a few attribute such as the name, type and id. This is no reason to make a context-agnostic component depend on a service used for a specific usage.

Reply:
Fixed